understanding/22/target-size-minimum.html

```diff
@@ -22,18 +22,16 @@ <h4>Target Size (Minimum)</h4>
 
-             <p>The size of the <a>target</a> for <a>pointer inputs</a> is at least 24 by 24 CSS pixels, except where:</p>
-
+             <p>The size of the <a>target</a> for <a>pointer inputs</a> is at least 24 by 24 <a>CSS pixels</a>, except where:</p>
             <ul>
-                <li><strong>Spacing:</strong> The <a>target offset</a> is at least 24 CSS pixels to every adjacent target;</li>
-                <li><strong>Inline:</strong> The target is in a sentence or block of text;</li>
-                <li><strong>Essential:</strong> A particular presentation of the target is <a>essential</a> or is legally required for the information being conveyed;</li>
+                <li><strong>Spacing:</strong> The target does not overlap any other target and has a <a>target offset</a> of at least 24 CSS pixels to every adjacent target;</li>
+                <li><strong>Equivalent:</strong> The function can be achieved through a different control on the same page that meets this criterion.</li>
+                <li><strong>Inline:</strong> The target is in a sentence, or is in a bulleted or numbered list, or its size is otherwise constrained by the line-height of non-target text;</li>
                 <li><strong>User agent control:</strong> The size of the target is determined by the user agent and is not modified by the author;</li>
-                <li><strong>Equivalent:</strong> The function can be achieved through a different control on the same page that has an area of at least 24 by 24 CSS pixels.</li>
+                <li><strong>Essential:</strong> A particular presentation of the target is <a>essential</a> or is legally required for the information being conveyed;</li>
             </ul>
-            <p class="note">Targets that allow for values to be selected spatially based on position within the target are considered one target for the purpose of the Success Criterion. Examples include sliders, color pickers displaying a gradient of colors, or editable areas where you position the cursor.</p>
-            <p class="note">Are there issues with internationalization when describing inline links?</p>
-            <p class="note">Are there issues with pop-over content overlapping targets triggering failures?</p>
+            <p class="note">Targets that allow for values to be selected spatially based on position within the target are considered one target for the purpose of the success criterion. Examples include sliders with granular values, color pickers displaying a gradient of colors, or editable areas where you position the cursor.</p>
+            <p class="note">For inline targets the line-height should be interpreted as perpendicular to the flow of text. For example, in a language displayed top to bottom, the line-height would be horizontal.</p>
             <dl><dt class="new"><dfn data-lt="target offsets">target offset</dfn></dt>
             <dd class="new">
                 <p class="change">New</p>
```

Comment:
The Target offset definition has also changed since understanding was written and needs an update

Reply:
I've copied it in from the normative text. (Noting that this version doesn't appear in the published docs.)